wire-ios-sync-engine/Source/SessionManager/SessionManager.swift

```diff
@@ -898,6 +907,13 @@ public final class SessionManager: NSObject, SessionManagerType {
         }
     }
 
+    //clears cache directory
+    fileprivate func clearCacheDirectory() {
+        guard let cachesDirectoryPath = cachesDirectory else { return }
+            let manager = FileManager.default
+            try? manager.removeItem(at: cachesDirectoryPath)
+    }
+
```

Comment:
A couple things:
- The comment is not adding any more info that I can't already get from the name, you can delete it.
- This doesn't need to be `fileprivate`, `private` is fine. 
- The indentation is incorrect